builder.js

```diff
@@ -118,7 +118,7 @@ Builder.prototype.build = function build(spec, callback) {
         // already been downloaded
         //
         cleanup();
-        writeStream.end({ eventType: 'complete' }, callback);
+        writeStream.end({ eventType: 'complete', message: 'Assets build completed' }, callback);
```

Comment:
nit: "Assets" here is a little misleading no? How about `message: 'carpenterd build completed'`?